src/SFA.DAS.VacancyServices.Functions/Infrastructure/StorageQueueService.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Queue;
+
+namespace SFA.DAS.VacancyServices.Functions.Infrastructure
+{
+    public class StorageQueueService
```

Comment:
Should this implement an interface?  And are there unit tests for it?

Reply:
I don't need an interface yet, so I avoided it. I kind of like to code only as much as it is required. I don't think there can be unit tests on this, we could write integration tests but is that even necessary?

Reply:
Why can't there be unit tests?